bin/anvi-run-ncbi-cogs

```diff
@@ -33,7 +34,8 @@ pp = terminal.pretty_print
 @time_program
 def main(args):
     cogs = COGs(args)
-    cogs.process()
+    aa_file = args.aa_file or None
+    cogs.process(aa_sequences_file_path=aa_file)
```

Comment:
The process function might become confusing now however as the case where "aa_sequences_file_path" and "--aa-file" arguments are both passed may be ambiguous. What was the role of this argument prior to my commit? I didn't identify it as doing anything other than allowing for cogs to consider a file input directly, thus my use of it. My solution to this comment would be to add a class variable that is set to the "--aa-file" argument in init, and add a check if both the argument and the class variable are provided, error out. Would that work?

Reply:
Ah, yes, that is indeed confusing. I'm very sorry I missed this.

I now realize the `aa_sequences_file_path` is produced by the class when a contigs-db is provided because BLAST or DIAMOND must be called to run on _that_ file. What you did is the right solution, and please ignore my comment on this item :)

Reply:
I will have to come back to this later to change the design a little so everything is a bit more tidy, but obviously that is independent of your PR. Thanks for your patience.